schemas/json/layout/layout.schema.v1.json

```diff
@@ -387,6 +388,30 @@
         }
       }
     },
+    "imageComponent": {
+      "properties": {
+        "image": {
+            "type": "object",
+          "title": "Image properties",
+          "description": "Set of options for image field.",
+          "properties": {
+              "src": {
+                  "title": "Image source",
+                  "description": "",
+                  "type": "object",
+                  "additionalProperties": true
+              },
+              "width": {
+                  "type":"string"
+              },
+              "align": {
+                  "type":"string",
+                  "enum": ["center", "right", "left"]
```

Comment:
Enum list is incorrect, the align layout is done with flexbox. You can see implementation (https://github.com/Altinn/altinn-studio/blob/master/src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/ImageComponent.tsx#L20) is using `GridJustification` from material-ui package (https://github.com/mui-org/material-ui/blob/v4.12.3/packages/material-ui/src/Grid/Grid.d.ts#L19), so this list should instead be 

```json
["flex-start", "center", "flex-end", "space-between", "space-around", "space-evenly"]
```

- [ ] update enum list